src/loaders/lottie/tvgLottieModel.h

```diff
@@ -676,6 +691,7 @@ struct LottieComposition
     Array<LottieInterpolator*> interpolators;
     Array<LottieFont*> fonts;
     Array<LottieSlot*> slots;
+    std::map<const char*,LottieMarker*,StrCompare> markers;
```

Comment:
Should we really have `std::map` for this?
Using 

`StrCompare` only provides a comparison function, I think we don't have to use in struct.
We can express like?:
```cpp
// Matching parameters and helper functions into `LottieMarker`
Array<LottieMarker*> markers;
```

Reply:
Please double-check binary size as well if you use new library.

```sh
cd ./build/src
strip libthorvg.so
size libthorvg.so
# compare output size with latest version of thorvg binary
```

Reply:
Resolved. But I think It's common for modern computers to have gigabytes or even terabytes of storage. Why we care about binary size? Just curious.😂

Reply:
@fall2019 ThorVG has been also considered for the web module, even for small-sized embedded IoT devices.
@tinyjin maintainers can do this, contributors don't need to be bothered by this.

Reply:
Ok, @fall2019 Never mind the binary size check!
But just fyi, the size is really important in the reason Hermet said ;)

Reply:
Yeah. That makes sense.